Kernel/Devices/HID/I8042Controller.cpp

```diff
@@ -109,7 +109,21 @@ UNMAP_AFTER_INIT ErrorOr<void> I8042Controller::detect_devices()
         TRY(do_wait_then_write(I8042Port::Command, I8042Command::WriteConfiguration));
         configuration &= ~I8042ConfigurationFlag::FirstPS2PortInterrupt;
         configuration &= ~I8042ConfigurationFlag::SecondPS2PortInterrupt;
+
+        // Note: We enable first port translation, because later on we set the keyboard
+        // to work with scan code 2 if the BIOS was not setting this for us already,
+        // but we rely on compatbility feature of the i8042 to send scan codes of set 1,
+        // because on the QEMU microvm machine type, it became apparent that the BIOS was
+        // not setting the i8042 controller to function as expected. To ensure that
+        // the controller is always outputting correct scan codes, set it to scan
+        // code 2 and enable first port translation to ensure all scan codes are
+        // translated to scan code set 1. This is the expected behavior when using
+        // SeaBIOS, but on qboot (the BIOS for the QEMU microvm machine type),
+        // the firmware doesn't take care of this so we need to do this ourselves.
+        configuration |= 0x40;
```

Comment:
I've tried to remove all magic constants from the I8042 code, would it be possible to assign an enum to both `0x40` and `0x2`?